TLSharp.Core/Network/Exceptions.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+namespace TLSharp.Core.Network
```

Comment:
I don't think we need this new file "Exceptions.cs", each exception that is in this file is now already in its own C# file in the master branch, if we include this they would be duplicated